ios/samples/gltf-viewer/project.yml

```diff
@@ -17,11 +17,20 @@ targets:
               optional: true
         templates:
             - FilamentApp
-        settings:
-            base:
-                OTHER_LDFLAGS: ["-lgltfio_core", "-luberarchive", "-limage", "-lstb", "-lbasis_transcoder",
-                                "-lzstd", "-lcamutils", "-ldracodec", "-lmeshoptimizer", "-lviewer", "-lcivetweb",
-                                "-luberzlib"]
+        dependencies:
+            - framework: $(SRCROOT)/../../../out/ios-release/filament/lib/libgltfio_core.xcframework
```

Comment:
Absolutely, it can be done - just tried to follow the current setup where common libraries are defined in `app-template.yml` and any specific (used by only one or a few examples) are defined in each example `project.yml` file.